doc/user-guide/hierarchical-data.rst

```diff
@@ -119,33 +117,36 @@ We can see the whole tree by printing Abe's node or just part of the tree by pri
 .. ipython:: python
 
     abe
-    homer
+    abe["Homer"]
 
-We can see that Homer is aware of his parentage, and we say that Homer and his children form a "subtree" of the larger Simpson family tree.
 
 In episode 28, Abe Simpson reveals that he had another son, Herbert "Herb" Simpson.
-We can add Herbert to the family tree without displacing Homer by :py:meth:`~DataTree.assign`-ing another child to Abe:
+We can add Herbert to the family tree without displacing Homer by :py:meth:`~xarray.DataTree.assign`-ing another child to Abe:
 
 .. ipython:: python
 
-    herbert = DataTree(name="Herb")
-    abe.assign({"Herbert": herbert})
+    herbert = xr.DataTree(name="Herb")
+    abe = abe.assign({"Herbert": herbert})
+    abe
+
+    abe["Herbert"].name
+    herbert.name
 
 .. note::
-   This example shows a minor subtlety - the returned tree has Homer's brother listed as ``"Herbert"``,
-   but the original node was named "Herbert". Not only are names overridden when stored as keys like this,
+   This example shows a subtlety - the returned tree has Homer's brother listed as ``"Herbert"``,
+   but the original node was named "Herb". Not only are names overridden when stored as keys like this,
    but the new node is a copy, so that the original node that was reference is unchanged (i.e. ``herbert.name == "Herb"`` still).
```

Comment:
```suggestion
   but the new node is a copy, so that the original node that was referenced is unchanged (i.e. ``herbert.name == "Herb"`` still).
```